src/formats/roam/utils.ts

```diff
@@ -8,10 +8,11 @@ const windowsTrailingRe = /[\. ]+$/;
 const startsWithDotRe = /^\./; // Regular expression to match filenames starting with "."
 const squareBracketOpenRe = /\[/g; // Regular expression to match "["
 const squareBracketCloseRe = /\]/g; // Regular expression to match "]"
+const ReplaceNoDir = '_';
```

Comment:
Can you please explain what this change is doing?

Reply:
I forgot to remove it - I was going to set it up to have a replacement character in a dropdown list in the options for importing, so any illegal filename characters would be replaced by it. For now I'll take it out (maybe in a future version)